network-api/networkapi/utility/templatetags/adminsortable_tags_custom.py

```diff
@@ -44,10 +51,13 @@ def render_object_rep(
     obj,
     forloop
 ):
-    context.update({'object': obj, 'forloop': forloop})
     tmpl = template.loader.get_template(
         '{app_name}/adminsortable_objects_custom.html'.format(
             app_name=context.get('opts').app_label
```

Comment:
this is still working?

Reply:
The trick was to pass `opts` along from context in every level of the template render.